.github/CODEOWNERS

```diff
@@ -468,7 +468,7 @@ tests/sentry/api/endpoints/test_organization_dashboard_widget_details.py     @ge
 /tests/sentry/auth/test_superuser.py                                    @getsentry/enterprise
 /tests/sentry/middleware/test_staff.py                                  @getsentry/enterprise
 /tests/sentry/tasks/integrations/github/                                @getsentry/enterprise
-/tests/sentry/models/releases/                                          @getsentry/alerts-notifications
+/tests/sentry/models/releases/                                          @getsentry/replay-frontend
```

Comment:
I realize this is _probably_ not the right team but I am not sure which one you are using

Reply:
```suggestion
/tests/sentry/models/releases/                                          @getsentry/replay-backend 
```